scripts/circle_deploy.sh

```diff
@@ -64,3 +56,21 @@ else
  echo "...publishing"
  npm publish
 fi
+
+#
+# gh-pages
+#
+echo "...deploying gh-pages"
+git push -f $CIRCLE_BRANCH:gh-pages
+git checkout gh-pages
+gulp docs
+git add .
+git add -f docs/build
+
+if [[ -n $(git status --porcelain) ]]; then
+  echo "...starting push, gh-pages is dirty after build"
+  git commit -n -m "deploy gh-pages by $CIRCLE_USERNAME [ci skip]"
+  git push origin gh-pages
```

Comment:
The checked out gh-pages here is forced pushed from master on ln 64.  So, the local copy is the latests remote copy.  A rebuild should force push master to gh-pages then checkout gh-pages again.  We'll see how it goes.